external/js/kyber/package.json

```diff
@@ -25,33 +26,54 @@
   "author": "DEDIS",
   "license": "AGPL-3.0",
   "homepage": "https://github.com/dedis/cothority",
+  "files": [
+    "dist/",
+    "doc/",
+    "index.html"
+  ],
   "dependencies": {
     "bn.js": "^4.11.8",
```

Comment:
Should these be updated now?

Reply:
No, _dist_ contain the actual library (transpiled js files and tyoes definition files), _doc_ the documentation and _index.html_ I guess it's good to have the example in npm.

Reply:
I meant the dependencies, should they be updated?

Reply:
Erf my bad. _bn.js_ is on the last version and you're right for _elliptic_, there is a minor update.